classes/person.rb

```diff
@@ -1,16 +1,28 @@
-class Person
+require_relative '../decorators/nameable'
+
+class Person < Nameable
+  attr_accessor(:name, :age, :rentals, :parent_permission)
+  attr_reader(:id)
```

Comment:
- [optional] please if you need only a getter method you can use only `attr_reader`. Likewise, if you need only a setter method, you will use only `attr_writer`. Or you can make it even shorter to avoid confusion by using: `attr_accessor :name, :age, :rentals` .. nice job